_beats/libbeat/scripts/Makefile

```diff
@@ -51,7 +51,7 @@ GOIMPORTS=goimports
 GOIMPORTS_REPO?=github.com/elastic/beats/vendor/golang.org/x/tools/cmd/goimports
 GOIMPORTS_LOCAL_PREFIX?=github.com/elastic
 GOLINT=golint
-GOLINT_REPO?=github.com/golang/lint/golint
+GOLINT_REPO?=golang.org/x/lint/golint
```

Comment:
I wonder why?  These should be the same

Reply:
APM Server overrides [GOLINT_REPO](https://github.com/elastic/apm-server/blob/4aa57602fb2fc7e0bd887705a17a915d0007cb73/Makefile#L10) to [vendor/github.com/golang/lint/golint](https://github.com/elastic/apm-server/tree/4aa57602fb2fc7e0bd887705a17a915d0007cb73/vendor/github.com/golang/lint/golint), which is still vendored appropriately.

Reply:
agreed, just curious